### PR TITLE
Breakout Room 9: Update main.py

### PR DESCRIPTION
**Pull request by tgoslee**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -1 +1,23 @@
...
+    noun = input("Choose a noun: ")
+    plural_noun = input("Choose a plural noun: ")
+    second_noun = input("Choose a noun: ")
+    place = input("Name a place: ")
+    adjective = input("Choose an adjective (Describing word): ")
+    third_noun = input("Choose a noun: ")
+// Displays the story based on the users input
+    print ("------------------------------------------")
+    print ("Be kind to your",noun,"- footed", plural_noun)
+    print ("For a duck may be somebody's", seond_noun,",")
```
Comment (buslakj) [inline main.py:15]: There is a typo for the second noun variable name.

File: main.py [hunk 2]
```diff
@@ -1 +1,23 @@
...
+    print ("Be kind to your",plural_noun,"in",place)
+    print ("Where the weather is always",adjective,".")
+    print ()
+    print ("You may think that is this the",third_noun,",")
+    print ("Well it is.")
+    print ("------------------------------------------")
+// Loop back to "loop = 1"
+    loop = loop + 1
```
Comment (buslakj) [inline main.py:23]: Syntax for comments is incorrect. Comments need to start with #.

File: main.py [hunk 3]
```diff
@@ -1 +1,23 @@
 #Mad Libs Generator Project
+//Loop back to this point once code finishes
+loop = 1
+while (loop < 9):
```
Comment (buslakj) [inline main.py:4]: Consider revisiting the requirements in regards to how many times this loop is ran. This is only being ran 8 times. We suggest starting with 0 and change it to "<=" instead of "<". Please consider the loop variable, is it functioning as intended?

File: main.py (hunk 2 above)
Comment (buslakj) [inline main.py:23]: By using functions we can separate the input and display. Please review the single responsibility principal.

File: main.py [hunk 4]
```diff
@@ -1 +1,23 @@
...
+    print ("------------------------------------------")
+    print ("Be kind to your",noun,"- footed", plural_noun)
+    print ("For a duck may be somebody's", seond_noun,",")
+    print ("Be kind to your",plural_noun,"in",place)
+    print ("Where the weather is always",adjective,".")
+    print ()
+    print ("You may think that is this the",third_noun,",")
+    print ("Well it is.")
+    print ("------------------------------------------")
+// Loop back to "loop = 1"
```
Comment (buslakj) [inline main.py:22]: Please consider reviewing and correcting formatting using an f-string.

File: main.py [hunk 5]
```diff
@@ -1 +1,23 @@
...
+//Loop back to this point once code finishes
+loop = 1
+while (loop < 9):
+// All the questions that the program asks the user
+    noun = input("Choose a noun: ")
+    plural_noun = input("Choose a plural noun: ")
+    second_noun = input("Choose a noun: ")
+    place = input("Name a place: ")
+    adjective = input("Choose an adjective (Describing word): ")
+    third_noun = input("Choose a noun: ")
```
Comment (buslakj) [inline main.py:11]: Consider placing a guard clause for invalid input and error handling.

File: main.py [hunk 6]
```diff
@@ -1 +1,23 @@
...
+// Displays the story based on the users input
+    print ("------------------------------------------")
+    print ("Be kind to your",noun,"- footed", plural_noun)
+    print ("For a duck may be somebody's", seond_noun,",")
+    print ("Be kind to your",plural_noun,"in",place)
+    print ("Where the weather is always",adjective,".")
+    print ()
+    print ("You may think that is this the",third_noun,",")
+    print ("Well it is.")
+    print ("------------------------------------------")
```
Comment (buslakj) [inline main.py:21]: It may be good to add a feature to allow the user to stop playing and to count how many times they have played out of the 10 trials.

File: main.py [hunk 7]
```diff
@@ -1 +1,23 @@
 #Mad Libs Generator Project
```
Comment (buslakj) [inline main.py:1]: Please try adding testing for edge cases

File: main.py (hunk 5 above)
Comment (buslakj) [inline main.py:11]: Could we place these into a dictionary and iterate through in order to create the string?

File: main.py [hunk 8]
```diff
@@ -1 +1,23 @@
...
+loop = 1
+while (loop < 9):
+// All the questions that the program asks the user
+    noun = input("Choose a noun: ")
+    plural_noun = input("Choose a plural noun: ")
+    second_noun = input("Choose a noun: ")
+    place = input("Name a place: ")
+    adjective = input("Choose an adjective (Describing word): ")
+    third_noun = input("Choose a noun: ")
+// Displays the story based on the users input
```
Comment (buslakj) [inline main.py:12]: One feature could be added here to display the line up to the inputted word for the user to see and build on the display after every input.

File: main.py [hunk 9]
```diff
@@ -1 +1,23 @@
 #Mad Libs Generator Project
+//Loop back to this point once code finishes
+loop = 1
```
Comment (buslakj) [inline main.py:3]: The variable name loop is confusing, try using counter to make this code more readable.

File: main.py (hunk 3 above)
Comment (buslakj) [inline main.py:4]: Why is this in parenthesis?
